src/Compilers/VisualBasic/Test/Symbol/SymbolsTests/CustomModifiersTests.vb

```diff
@@ -2631,695 +2631,5 @@ BC40001: 'Public Overrides Function M(x As (Object, Object)) As (a As Object, b
 
         End Sub
 
-        <Fact()>
```

Comment:
It would be convenient if in the future, a change like this which moves a chunk of code to another file could break the change down into:

1. a commit that moves the code without modification
2. a commit that modifies the code

Reply:
> It would be convenient if in the future, a change like this which moves a chunk of code to another file could break the change down into: ...

I didn't do that because I realized that I already added the tests into a different file when I opened the issue. Basically, I discovered that they are there when they failed. In this particular case, I don't believe there was a particular value in treating the deletion as a move. Treating the tests as brand new would be preferred, in my opinion.